website/models.py

```diff
@@ -5,12 +5,13 @@
 class Image(models.Model):
     image = models.ImageField(blank=True)
     user = models.ForeignKey(
-        User, related_name='images',
+        User,
+        related_name='images',
         on_delete=models.CASCADE
     )
 
 
 class Profile(models.Model):
```

Comment:
Use AbstractUser

Reply:
AbstractUser added into models.py and update in other files